ios/RCTWKWebView/RCTWKWebViewManager.m

```diff
@@ -17,24 +17,19 @@ @implementation RCTWKWebViewManager
 {
   NSConditionLock *_shouldStartLoadLock;
   BOOL _shouldStartLoad;
-  WKProcessPool *_processPool;
```

Comment:
We don't seem to need a hold to this ```WKProcessPool``` reference, as we are passing it to the ```RCTWKWebView``` instance created on ```- (UIView *)view```.

The issue with holding the instance as we are, is that if the app is restarting and all components are destroyed and re-created, the destruction of this ```RCTWKWebViewManager``` instance crashes on iOS 11 when this ```WKProcessPool``` gets disposed.

I don't know if the fact that an ```RCTWKWebView``` is holding a reference could be affecting it at all.